js/controller.js

```diff
@@ -1244,7 +1244,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.renderSkin();
         this.mb.publish(OO.EVENTS.PAUSE);
         if (selectedContentData.clickedVideo.embed_code){
-          this.mb.publish(OO.EVENTS.SET_EMBED_CODE, selectedContentData.clickedVideo.embed_code);
+          this.mb.publish(OO.EVENTS.SET_EMBED_CODE, selectedContentData.clickedVideo.embed_code,
+                          this.state.playerParam);
```

Comment:
No it does not because the logic for discovery auto plays is covered in the onPlayed method